TidepoolOnboardingKitUI/View Controllers/PrescriptionReviewUICoordinator.swift

```diff
@@ -0,0 +1,376 @@
+//
+//  PrescriptionReviewUICoordinator.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Anna Quinlan on 6/18/20.
+//  Copyright © 2020 Tidepool Project. All rights reserved.
+//
+
+import os.log
```

Comment:
this looks like a cut and paste of the current code. If that isn't the case, let me know and I'll look at it closely.

Reply:
Yes, this was mostly just moving this file from the `TidepoolService` repo to this repo. There is some temporary "glue" code (all of the delegates at the top) to connect it into the onboarding workflow, but most of that has gone away with the next set of PRs anyhow. So, yes, mostly cut and paste.

Reply:
I ditto Nate's comment.